src/micro/attributes.html

```diff
@@ -83,7 +83,7 @@
         // NOTE: never allow 'class' to be set in hostAttributes
         // since shimming classes would make it work
         // inconsisently under native SD
-        if (!this.hasAttribute(n) && (n !== 'class')) {
+        if (!this.hasAttribute(n) && (n !== 'class') && this._config[n] === undefined) {
```

Comment:
@sorvell @cdata 

Ok, spent a little time to recall what where we ended with this.  These changes are what was in my stash from when we looked at this last.  Everything is chill except for this one change here.

Best I can (re-)work out, [this test](https://github.com/Polymer/polymer/blob/master/test/unit/attributes.html#L286) fails without the `&& this._config[n] === undefined` here.  That test tests that setting a `hostAttribute` on a host, which is then deserialized and bound down into an attribute of a child which itself also defines a `hostAttribute` with a different value, only results in the child property observer only being called once with the host value.  It fails not because the ending value is wrong, but because the callback is called more than once.  This is because of the queueing nature of `attributeChangedCallback` when `setAttribute` is called from the context of a `createdCallback` (on Chrome only); the host's setAttribute of the overriding value is queued, and runs after the child readies itself with its own attribute value. (Tangentially, it also fails in an identical way on polyfill where `setAttribute` runs the callback synchronously for different reasons, but the change here resolves the issue for both)

Next steps are:
* Do we even want to fix this?  Best I can tell the setup is totally contrived and worst case it arrives at the same values, just with an extra observer call.  We could just "fix the test".
* If we do want to fix this, we have a layering violation here between configuration (standard) & attributes (micro), so we'd need to deal with that.
* Last, I just noted that the `n` here is an attribute name, so it looks wrong that `n` is used as a key to `_config` which is property-named.  Probably requires adding a case-map call here, which is slightly worse for perf.

Seems like that makes me lean towards just fixing the test and removing the change on this line.  Let's discuss tomorrow.